be/src/olap/compaction.cpp

```diff
@@ -127,8 +127,11 @@ OLAPStatus Compaction::do_compaction_impl(int64_t permits) {
     int64_t now = UnixMillis();
     if (compaction_type() == ReaderType::READER_CUMULATIVE_COMPACTION) {
         _tablet->set_last_cumu_compaction_success_time(now);
+        _tablet->update_cumulative_compaction_score(_tablet->calc_compaction_score(CompactionType::CUMULATIVE_COMPACTION));
```

Comment:
I found every call of  `update_cumulative_compaction_score()` 's parameter is `calc_compaction_score()`, what's reason to do this?